### PR TITLE
Make tmdb api key configurable

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: Emby.Server.Implementations/ConfigurationOptions.cs [hunk 1]
```diff
@@ -19,7 +19,8 @@ public static class ConfigurationOptions
             { FfmpegProbeSizeKey, "1G" },
             { FfmpegAnalyzeDurationKey, "200M" },
             { PlaylistsAllowDuplicatesKey, bool.FalseString },
-            { BindToUnixSocketKey, bool.FalseString }
+            { BindToUnixSocketKey, bool.FalseString },
+            { TmdbApiKeyKey, "4219e299c89411838049ab0dab19ebd5" }
```
Comment (cvium) [inline Emby.Server.Implementations/ConfigurationOptions.cs:23]: Ideally the providers should be separate plugins.

**Review by crobibero — APPROVED** [review]

**Review by dkanada — COMMENTED** [review]
This should really be using the existing plugin configuration pages like the other providers that allow custom API keys.

**Review by cvium — CHANGES_REQUESTED** [review]
The goal is to split providers into plugins. This PR is not conducive to that goal.

**Comment by dkanada** [comment]
If this moves to the TMDB plugin configuration though, it would be fine. Either it already exists (about half of the bundled plugins implement IPluginConfiguration) or it could be added relatively easily.

**Comment by joshuaboniface** [comment]
Yea I think this can be closed with the removal of TMDB in #4530. Gonna close it.

**Pull request closed without merging**